packages/preprocess/src/process-queries.cjs

```diff
@@ -261,6 +267,15 @@ const createDefaultProps = function (filename, componentDevelopmentMode, duckdbQ
 			}, 5000);
 		};
 
+		const longQueryNotifier = (info) => {
+			toasts.add({
+				id: Math.random(),
+				title: info.id,
+				message: \`Query \${info.id} is loading...\`,
+				status: 'info'
```

Comment:
I think the behaviour we want on the toast isn't several toasts saying "query x is loading..." but rather just a single "loading..." toast that's present until all long-running queries are done.